### ISSUE TITLE
Search also bible text in bookmarks

### ISSUE THREAD

**Issue by nikkrossi** [AndBible/and-bible#3339]
<details><summary>Summary</summary>
<p>

**Title:** Suggestion to Enhance Bookmark Search Functionality  

**Summary:** The user suggests improving the bookmark search functionality by allowing searches to include the text of bookmarked scriptures, in addition to attached notes.  

**Key Points:**  
1. **Current Functionality:** Bookmark search is limited to notes attached to bookmarked verses.  
2. **Suggested Improvement:** Include the text of the scriptures in the bookmark search functionality for better usability.  
3. **Use Case:** This would help users find specific verses more efficiently within their tagged bookmarks, especially when they do not remember exact references but recall keywords.  
4. **Appreciation:** The user expresses gratitude for the app and its development.  

---  
*Summarized using  [[Summarize @ ChatGPT](https://chatgpt.com/g/g-tFjop0Fow-summarize)](https://chatgpt.com/g/g-tFjop0Fow-summarize)*

</p>
</details> 

As I have been using this bible for some serious study and learning through the Greek and Hebrew, I've found myself bookmarking using various tags. While I love the tags and being able to categorize what I'm bookmarking, the **Bookmark** page only allows you to search for notes that you written about your bookmarked verses

For example if I had 100 bookmarks in a **Tag** called "Love" and no notes written about those verses. One of the verses is 1 Cor. 13:4. Now lets say I don't remember where that is just that it contains "love" and "kind". While I could look that up in the larger **Find** function, which would bring me multiple results, I know I have bookmarked it and it would limit results if i searched through the tagged bookmarks. 

So I would like to go to **Bookmarks**, and instead of just being able to search notes that I've attached to my bookmarks. I would like the search to also include the text of the scripture. 

Thank you for all the hard work that you do and for bringing forth this wonderful resource.
God Bless!

![Screenshot_20241114_120128_Bible Study](https://github.com/user-attachments/assets/c4390448-157e-4d57-be4c-5cd521411666)

**Comment by tuomas2** [comment]
Good idea. I think this was discussed in the PR (this feature was contributed by someone) when the search feature was made.

**Comment by tuomas2** [comment]
Estimate: 4h
